content/docs/command-reference/remote/modify.md

```diff
@@ -758,6 +758,43 @@ more information.
 
 </details>
 
+<details>
+
+### Click for OSF
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify myremote url \
+        osf://odf.io/path/to/dir
+
+- `project` - [OSF](https://osf.io/) project name. You can see it in the `url` line
+  of your project page.
```

Comment:
> You can see it in the `url` line of your project page.

What does that mean @aerubanov? What is the project page? What is the url line (how is it different to the `url` remote option)? Thanks

Reply:
Also, is this optional or when would you need to provide a project name, if for `remote add` only a `url` is required?

Reply:
A project page is like a repository page on github, only in osf. The project page is similar to the repository page on github, only in osf. You can open it in your browser. And the address bar will be for example https://osf.io/abcd/. `abcd` is what you need to specify as a project parameter.

Reply:
No, the role user must specify all the `user`, `project` and `password` parmeters for proper work. Do I need to specify it somehow?

Reply:
Thanks for clarifying in docs and for the answers 🙂